code/espurna/config/hardware.h

```diff
@@ -2874,6 +2874,28 @@
     #define LIGHT_CH3_INVERSE   0
     #define LIGHT_CH4_INVERSE   0
 
+// -----------------------------------------------------------------------------
+// Lombex LX_BW09
+// https://www.amazon.com/Compatible-Automation-Dimmable-Equivalent-Changing/dp/B07587CNFV
+// -----------------------------------------------------------------------------
+#elif defined(LOMBEX_BW09)
```

Comment:
The current version requires the board name to be MANUFACTURER_DEVICE, so this should be "LOMBEX_LX_BW09". Change it also in the platformio.ino file.

Reply:
K, I'll keep that in mind. I might need to cancel this request altogether and submit a new change later on due to two developments:

1. I blew up the bulb (Started making a repeated cap squeak noise and GNC and VCC are now shorted). It's possible it isn't designed to be run on both the RGB channels and warm LEDS full blast for very long, I think I had them on overnight. Have you experienced any problems like this?

2. I just went to experiment with a different newer variant of these Lombex bulbs "Lombex Lux Nova" that just have warm and cold LEDS, and found that the model number is frustratingly exactly the same.